src/components/spaces/show/index.js

```diff
@@ -97,17 +93,17 @@ export default class SpacesShow extends Component {
 
   render() {
     const space = this.props.denormalizedSpace;
-    const sidebarIsViseable = !!space && (space.editableByMe || !_.isEmpty(space.description))
-    const canMakeMorePrivateModels = e.me.canMakeMorePrivateModels(this.props.me)
+    if (!space) { return <div className='spaceShow'></div> }
+
+    const sidebarIsViseable = space.editableByMe || !_.isEmpty(space.description)
+    const canBePrivate = !!space.organization_id || e.me.canMakeMorePrivateModels(this.props.me)
```

Comment:
I feel like this will come back to bite us, if we have free organizations without private functionality.